packages/lesswrong/server/emails/renderEmail.tsx

```diff
@@ -210,7 +214,7 @@ export async function generateEmail({user, to, from, subject, bodyComponent, boi
     user,
     to,
     from: fromAddress,
-    subject: taggedSubject,
+    subject: isEAForum ? subject : taggedSubject,
```

Comment:
The `taggedSubject` is a strange pattern to me. I think I've only seen it in the context of mailing lists? Anyway I don't really think it makes sense in the context of a website sending you emails, so it's possible we should change the default to remove it 🤷‍♀️

Reply:
Yep if we could guarantee the sender name will be set correctly I think it would be better to remove it